db/schema.rb

```diff
@@ -1084,6 +1115,7 @@
     t.text "round_results", size: :medium
     t.integer "total_number_of_rounds", null: false
     t.string "old_type", limit: 1
+    t.boolean "is_open", default: false, null: false
```

Comment:
I don't see a corresponding migration for this. Potential whoopsie from your other PR?

Reply:
yeah, that should probably go into the open round pr instead